<?xml version="1.0"?>
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="dynamic_slp">
                <param name="k1" value="1.0" />
                <param name="k2" value="1.0" />
                <param name="k3" value="1.0" />
                <param name="k4" value="1.0" />

                <param name="gamma" value="2.0" />
                <param name="theta_a" value="4*0.78539816339" />
                <param name="max_angular_velocity" value="0.5" />
                <param name="epsilon" value="0.5" />
                <param name="b" value="0.2" />

                <param name="look_ahead_dist" value="0.5" />
                <param name="k_o" value="0" />
                <param name="k_g" value="0.4" />
                <param name="k_w" value="0.1" />
                <param name="k_curv" value="0.00" />

                <param name="m" value="50.00" />
                <param name="I" value="0.0356" />
                <param name="r" value="0.1" />
                <param name="w" value="0.2335" />
                <param name="gearbox" value="12.52" />
                <param name="Kt" value="0.08" />
            </group>
        </group>
    </group>  
</launch>
